Example/Podfile.lock

```diff
@@ -120,4 +120,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ab2b14498bb09c9ba993d5484696be2421070207
 
-COCOAPODS: 1.0.1
+COCOAPODS: 1.1.0.rc.2
```

Comment:
I’ve pushed a commit that reverts to use CP 1.0.1, but the emojis don’t go away, I suspect that is because CP skips the build phase as it already exists.